packages/synapse-react-client/src/components/RowDataTable/RowDataTable.tsx

```diff
@@ -0,0 +1,93 @@
+import { SelectColumn } from '@sage-bionetworks/synapse-types'
+import { LabelLinkConfig } from '../CardContainerLogic'
+import { StyledTableContainer } from '../styled/StyledTableContainer'
+import { SynapseCardLabel } from '../GenericCard'
+
+export type RowDataTableProps = {
+  /** Array of string values corresponding to the data for a row */
+  rowData: string[]
+  /** The headers for the columns in the table */
+  headers: SelectColumn[]
+  /** List of column names to display as labels */
+  labels: string[]
+  /** Optional configuration for linking columns */
+  columnLinks?: LabelLinkConfig
+  /** key-value pair mapping to display in rows */
+  labelValueArray?: [string, string][]
```

Comment:
I don't think you will ever want to pass this prop, you will always create this data structure in the component
```suggestion
```